spec.bs

```diff
@@ -575,17 +574,35 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     [=parallel queue/enqueue steps|enqueue the following steps=] to |queue|:
     1. Let |compWinner| be the result of [=generate and score bids=] with |component|,
       |auctionConfig|, and |global|.
-    1. If |compWinner| is not failure or null, [=score and rank a bid=] with |auctionConfig|,
-      |compWinner|, |leadingBidInfo|, |desisionLogicScript|, and true.
+    1. If |compWinner| is not null:
+      1. Run [=score and rank a bid=] with |auctionConfig|, |compWinner|, |leadingBidInfo|,
+        |desisionLogicScript|, and true.
     1. Decrement |pendingComponentAuctions| by 1.
   1. Wait until |pendingComponentAuctions| is 0.
+  1. If |leadingBidInfo|'s [=leading bid info/leading bid=] is null, return null.
+  1. Let |winningComponentConfig| be |leadingBidInfo|'s [=leading bid info/auction config=].
+  1. Set |leadingBidInfo|'s [=leading bid info/auction config=] to |auctionConfig|.
+  1. Set |leadingBidInfo|'s [=leading bid info/component seller=] to |winningComponentConfig|'s
+    [=auction config/seller=].
+  1. Let « |topLevelSellerSignals|, topLevelReportResultBrowserSignals » be the result of running

```

Comment:
topLevelReportResultBrowserSignals isn't inside ||s -- where does this get used later?

Reply:
It doesn't; it's thrown away.

Reply:
Maybe prefix the variable with unused? Not sure the convention for specs, but that would make it more clear (for me, at least) that it's intentionally unused.